examples/CMakeLists.txt

```diff
@@ -56,6 +56,7 @@ add_example(rlfunctions)
 add_example(shaders_basic)
 add_example(core_vsync_fullscreen)
 add_example(core_msaa)
+add_example(core_multiwindow_multiplatform)
```

Comment:
shouldn't this one only be built if SUPPORT_SDL3 && SUPPORT_GLFW && SUPPORT_RGFW?
Or checking inside the example for which is supported with some ifdefs?